best-practices/require_pod_requests_limits/require_pod_requests_limits.yaml

```diff
@@ -9,9 +9,11 @@ metadata:
     policies.kyverno.io/subject: Pod
     policies.kyverno.io/description: >-
       As application workloads share cluster resources, it is important to limit resources 
-      requested and consumed by each pod. It is recommended to require 'resources.requests' 
-      and 'resources.limits.memory' per pod. If a namespace level request or limit is specified, 
-      defaults will automatically be applied to each pod based on the 'LimitRange' configuration.
+      requested and consumed by each Pod. It is recommended to require 'resources.requests' 
```

Comment:
should this be `resources.requests.memory`?

Reply:
maybe update to `It is recommended to require resource requests and limits per pod, especially for memory and CPU.`